docs/index.md

```diff
@@ -1,6 +1,6 @@
 # Welcome to OpenScPCA
 
-OpenScPCA is an open, collaborative project to analyze data from the [Single-cell Pediatric Cancer Atlas (ScPCA) Portal](https://scpca.alexslemonade.org/), which currently holds 500 samples from over 50 pediatric cancer types.
+OpenScPCA is an open, collaborative project to analyze data from the [Single-cell Pediatric Cancer Atlas (ScPCA) Portal](https://scpca.alexslemonade.org/), which currently holds over 700 samples from 55 pediatric cancer types.
```

Comment:
```suggestion
OpenScPCA is an open, collaborative project to analyze data from the [Single-cell Pediatric Cancer Atlas (ScPCA) Portal](https://scpca.alexslemonade.org/), which currently holds over 700 samples from over 50 pediatric cancer types.
```
Really minor but maybe let's keep this so it doesn't get out of date too soon?

Reply:
good call!